pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2635,4 +2635,12 @@ in {
 
   # 2022-10-04: Needs newer tasty-dejafu than (currently) in stackage
   rec-def = super.rec-def.override { tasty-dejafu = self.tasty-dejafu_2_1_0_0; };
+
+  # 2022-11-03: Builds with brick 1.0 which in turn works with text-zipper 0.12
+  # Other dependencies are resolved with doJailbreak for both swarm and brick_1_0
+  swarm = doJailbreak (super.swarm.override {
```

Comment:
It supports brick 1.3 as well which we ship in the package set, so use that instead.

Reply:
alright, so it only breaks with 1.4 I guess

Reply:
I am unable to locate brick_1_3 in hackage-packages.nix :(

Reply:
Because it’s not generated without intervention. You entered `brick == 1.0` in the `main.yaml` replace it with `brick == 1.3` and regenerate the hackage packages file.

Reply:
done!